dwave_networkx/drawing/chimera_layout.py

```diff
@@ -51,6 +51,11 @@ def chimera_layout(G, scale=1., center=None, dim=2):
         Number of dimensions. When dim > 2, all extra dimensions are
```

Comment:
Now ``scale`` is divided by ``fabric_scale`` in the ``chimera_node_placer_2d`` function  - should the docstring for ``scale`` in line 44 be updated?

Reply:
There is no net change in behavior here; I just needed to capture the value of `fabric_scale` for use in `normalize_size_and_aspect`.